frontend/src/main/components/Users/UsersTable.js

```diff
@@ -16,16 +14,35 @@ export default function UsersTable({ users}) {
         }
     }
 
+    function cellToAxiosParamsToggleDriver(cell){
+        return {
+            url: "/api/admin/users/toggleDriver", // going to have to add api/driver
+            method:  "POST",
+            params: {
+                id: cell.row.values.id
+            }
+        }
+    }
+
     // Stryker disable all : hard to test for query caching
     const toggleAdminMutation = useBackendMutation(
         cellToAxiosParamsToggleAdmin,
         {},
         ["/api/admin/users"]
     );
+
+    const toggleDriverMutation = useBackendMutation(
+        cellToAxiosParamsToggleDriver,
+        {},
+        ["/api/admin/users"] 
+
+    );
+
     // Stryker enable all 
```

Comment:
Please replace with 
```suggestion
    // Stryker restore all 
```

See: https://ucsb-cs156-s23.slack.com/archives/C050779M19P/p1685726928710669